ui/apps/platform/src/Containers/Collections/RuleSelector/RuleSelector.test.tsx

```diff
@@ -61,14 +61,14 @@ describe('Collection RuleSelector component', () => {
 
         render(<DeploymentRuleSelector defaultSelector={{ type: 'All' }} onChange={onChange} />);
 
-        await user.click(screen.getByLabelText('Select deployments by name or label'));
-        await user.click(screen.getByText('Deployments with names matching'));
+        await act(() => user.click(screen.getByLabelText('Select deployments by name or label')));
+        await act(() => user.click(screen.getByText('Deployments with names matching')));
```

Comment:
The user events called by `user.click()` _should_ already be wrapped internally and not require wrapping each one individually in `act`. I ran into a similar issue with this a while back where there were two separate versions of testing library dependencies installed that were conflicting.

It is possible that the [two major versions](https://github.com/stackrox/stackrox/pull/8314/files#diff-3a968206d6de2fecfc5dacd7d94bab7744c9f5d5c999a816164d95cbc135c316R3813-L3831) of `@testing-library/dom` in the yarn.lock file in this PR could be creating problems. The [docs warn about this](https://testing-library.com/docs/user-event/install/), but we aren't including that dependency directly in our package.json.